src/app/admin/_components/Event/TagList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function TagList({ tags, isLoading, error }) {
+  const tagItems =
+    tags?.map(tag => (
+      <div className="rounded-full bg-[#2194f3] px-4 py-2 text-other-white" key={tag.id}>
+        {tag.name}
+      </div>
+    )) || [];
```

Comment:
Nice that you've added key here.
Consider also a small improvement - fallback for `tags`. Eg:
```suggestion
export function TagList({ tags = [], isLoading, error }) {
  const tagItems =
    tags.map(tag => (
      <div className="rounded-full bg-[#2194f3] px-4 py-2 text-other-white" key={tag.id}>
        {tag.name}
      </div>
    ));
```

Reply:
done it